src/Controller/ProcessOverviewController.php

```diff
@@ -40,6 +40,8 @@ public function show(ProcessOverviewGroup $group, ProcessOverview $overview): Re
                     'Go to next page' => t('Go to next page'),
                     'Missing data' => t('Missing data'),
                     'Failed processes' => t('Failed processes'),
+                    'Viewing processes failed in' => t('Viewing processes failed in'),
```

Comment:
We're “Showing” processes in line 52, so I suggest


```suggestion
                    'Showing processes failed in' => t('Showing processes failed in'),
```

We're actually showing “process runs”, but it probably makes more sense to real people to (still) call them “processes”.

Reply:
I stole this and added it as a comment